.github/workflows/be_review_prs.yml

```diff
@@ -33,47 +25,54 @@ jobs:
           ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
           env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
 
+# Find Backend Labels, Approvals and Comments
       - name: Find Approval Required Comment
         uses: peter-evans/find-comment@v3
-        id: find-required-comment
+        id: find_backend_comment
         with:
           issue-number: ${{ github.event.pull_request.number }}
           body-includes: This PR requires approval from backend-review-group.
 
-      - name: Github event
-        run: echo "${{ github.event_name }}"
-
-      - name: Pull Request Review Teams
-        id: review_teams
-        run: |
-          REQUESTED_TEAMS=$(echo '${{ steps.pr_details.outputs.data }}' | jq -r '.teams[].slug')
-          echo "REQUESTED_TEAMS=$REQUESTED_TEAMS" >> $GITHUB_ENV
-
-      - run: echo "$REQUESTED_TEAMS"
+      - name: Find Approval  Comment
+        uses: peter-evans/find-comment@v3
+        id: find_backend_approval_comment
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body-includes: Backend-review-group approval confirmed.
 
-      - name: Add Assign team to Pull Request Review Comment
-        id: assign_team_review_comment
-        if: env.REQUESTED_TEAMS == ''
-        uses: peter-evans/create-or-update-comment@v3
+      - name: Find Ommitted Team Comment
+        uses: peter-evans/find-comment@v3
+        id: find_omit_comment
         with:
           issue-number: ${{ github.event.pull_request.number }}
-          body: "There were changes made to this pull request after the inital approval. Github API may haved unassigned your team from Reviewers. Please assign your team or backend-review-group to this pull request and re-run this check again."
+          body-includes: Omit backend approval
+
+      - name: Re-label require-backend-approval
+        if: |
+          steps.find_backend_comment.outputs.comment-id != '' &&
+          !contains(github.event.pull_request.labels.*.name, 'require-backend-approval')
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          number: ${{ github.event.pull_request.number }}
+          labels: |
+            require-backend-approval
 
-      - name: Fail if no teams are requested
-        if: env.REQUESTED_TEAMS == ''
+      - name: Verify require-backend-approval label
+        if: contains(github.event.pull_request.labels.*.name, 'require-backend-approval') && steps.find_backend_comment.outputs.comment-id == ''
         run: exit 1
 
-      - name: Check required reviewers
-        id: check_reviewers
+      - name: Verity omit-backend-approval label
+        if: contains(github.event.pull_request.labels.*.name, 'omit-backend-approval') && steps.find_omit_comment.outputs.comment-id == ''
```

Comment:
Fail if comment is not present and the omit-backend-approval is there. This is to prevent people from adding/removing labels